### ISSUE TITLE
Enhance Worklist Display - Task name & Task details

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#406]
```
At the moment the engine id's of work items are displayed in the select box. To 
be honest, an end-user doesn't care about the engine id. I think it'd be better 
to display task name in the list and the engine id in the details panel on the 
right.

Also, often there are cases were people get offered work items of the same type 
simultaneously just because they get offered to a whole unit of a department, 
e.g. 10 people. These 10 people have to work on these items but not everyone 
can work on every item because it wasn't his desk where the documents relating 
to the work item landed. So to choose the right work item one must know basic 
details about the work item or the case itself, e.g. the person of the case 
that this work item is related to. At the moment it is impossible to see such 
information without accepting and starting a work item and then finding out it 
was the wrong one and reallocate it to the right person. This is of course a 
waste of time.

What would be the best way to store and get such information to display in the 
worklist? Obviously not in input task variables because they just get filled 
after task start. Maybe an extra XQuery enabled "Description" field which is 
associated to a task so that one could construct this description with net 
parameters?

What do you think?

NB: I noticed that these issues get filed as defects, why do normal users not 
have the ability to submit enhancements?
```

Original issue reported on code.google.com by `neothema...@googlemail.com` on 31 Aug 2010 at 1:20

**Comment by yawlfoundation** [comment]
```
Changing to Enhancement (this definitely is).
```

Original comment by `monsieur...@gmail.com` on 31 Aug 2010 at 3:06
* Added labels: **Type-Enhancement**
* Removed labels: **Type-Defect**

**Comment by yawlfoundation** [comment]
```
implemented for 2.2beta
```

Original comment by `yawl.mic...@gmail.com` on 13 May 2011 at 9:50
* Changed state: **Fixed**
